pkg/graveler/uncommitted_diff_iterator.go

```diff
@@ -109,5 +110,5 @@ func (d *uncommittedDiffIterator) Err() error {
 }
 
 func (d *uncommittedDiffIterator) Close() {
```

Comment:
Need to close the committedList as well.

Reply:
Thanks